modules/homepage/src/main/frontend/src/themePage/index.jsx

```diff
@@ -70,6 +70,7 @@ class Main extends React.Component {
         return (
           <Route
             path={route["lfs:targetURL"]}
+            exact={route["lfs:exactURLMatch"] || false}
```

Comment:
```suggestion
            exact={Boolean(route["lfs:exactURLMatch"])}
```
Prefer `Boolean()` to `|| false`, because it's more explicit as to why it's necessary.

Reply:
I didn't do that initially because `Boolean("false") == true`, while I expected `exact={"false"}` to actually be false. Turns out that `exact={"false"}` actually results in a "Failed prop type" error. I'll do as you suggest.